tests/dialogs/src/index.ts

```diff
@@ -0,0 +1,303 @@
+import { cardAttachment, MessageActivity } from '@microsoft/teams.api';
+import { App, HttpPlugin } from '@microsoft/teams.apps';
+import {
+  Card,
+  ICard,
+  SubmitAction,
+  TaskFetchAction,
+  TaskFetchData,
+  TextInput,
+} from '@microsoft/teams.cards';
+import { DevtoolsPlugin } from '@microsoft/teams.dev';
+import dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config({
+  path: path.join(__dirname, '..', 'env', '.env.local'),
+});
+
+const httpPlugin = new HttpPlugin();
+
+if (!process.env['BOT_ENDPOINT']) {
+  console.warn('No remote endpoing detected. Using webpages for dialog will not work as expected');
+}
+
+httpPlugin.get('/', (_req, res) => {
```

Comment:
this is built in now via `app.tab` and the tab gets hosted at url `/tabs/{name}`